rust/ql/test/library-tests/type-inference/type-inference.expected

```diff
@@ -2970,3 +3973,93 @@ inferType
 | main.rs:1928:41:1928:59 | ...::Foo {...} |  | main.rs:72:5:72:21 | Foo |
 | main.rs:1944:5:1944:15 | ...::f(...) |  | {EXTERNAL LOCATION} | trait Future |
 testFailures
+| main.rs:2:5:2:20 | f.write_str(...) | Unexpected result: method=write_str |
```

Comment:
You've allowed some test failures without updating the annotations in the `.rs` file.  This isn't a good state to leave things in as the annotations are now misleading.  Is there a plan to fix these?